webapp/eslint.config.mjs

```diff
@@ -0,0 +1,39 @@
+import globals from "globals";
+import esLintPluginReact from "eslint-plugin-react";
+import esLintPluginReactHooks from "eslint-plugin-react-hooks";
+
+export default [
```

Comment:
Have you manually tested the main UI flows, e.g. branch page, workbench and uploading a screenshot?

Feel free to resolve this comment yourself if that's already been done, there's no automated UI tests so we will need to manually verify that this doesn't introduce an issue somewhere (I'm thinking of the file name changes from .js to .jsx as a potential source of a problem)

Reply:
It did break for me until I made the webpack changes. 
When I resolved the issues with webpack, I just did some spot checking on the UI. I didn't go through a full testing flow. I will do that tomorrow

Reply:
I've done a full test this morning including:
-  adding translations
- searching for translations in a repo
- changing translation statuses
- uploading a screenshot to a branch
-  viewing a screenshot
-  deleting a screenshot